src/ontology/edits/oeo-tasks.omn

```diff
@@ -0,0 +1,709 @@
+    
+    SubClassOf: 
+        OEO_00390035
+    
+    
+Class: OEO_00390007
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determination process for wind characteristics of a region is a process that follows a determination methodology for wind characteristics to gain knowledge of the specific wind characteristics of a predefined region.",
+        rdfs:label "determination process for wind characteristics of a region"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000015>,
+        OEO_00390026 some OEO_00390002
+    
+    
+Class: OEO_00390014
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Account for air density change is an action specification that describes how variations in air density due to changes in altitude, temperature, and pressure are incorporated into the wind resource calculations.",
+        rdfs:label "account for air density change"@en
+    
+    SubClassOf: 
+        OEO_00390035
+    
+    
+Class: OEO_00390015
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determine wind speed frequency distribution is an action specification that describes the process of calculating the frequency of different wind speed intervals over a set period.",
+        rdfs:label "determine wind speed frequency distribution"@en
+    
+    SubClassOf: 
+        OEO_00390035
+    
+    
+Class: OEO_00390016
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determine seasonal/diurnal variability is an action specification that describes the process of identifying variations in wind characteristics across different seasons and times of day.",
+        rdfs:label "determine seasonal/diurnal variability"@en
+    
+    SubClassOf: 
+        OEO_00390035
+    
+    
+Class: OEO_00390017
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determine the theoretical potential / WPD is an action specification that describes how Wind Power Density (WPD) is calculated to estimate the theoretical wind power potential based on wind speed and air density.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Determine the theoretical wind power potential / wind power density is an action specification that describes how wind power density (WPD) is calculated to estimate the theoretical wind power potential based on wind speed and air density.",
```
Please add to to do list in Issue:
We have something like theoretical potential and maybe also wind power density in the OEO. We should add axioms in a later step (OEO_00140063).